### DISCUSSION TITLE
Other Code Editors

### DISCUSSION THREAD

**Discussion by Darktechnodream** [mrdbourke/pytorch-deep-learning#625] [General]
My question is that can I use Pycharm or do I need to us Cloab?

**Comment by otrturn** [comment]
PyCharm is an integrated development environment used for programming in Python. It is a product from JetBrains just like Intellij. Colab is a hosted Jupyter Notebook service that requires no setup to use and provides free access to computing resources, including GPUs and TPUs. Colab is especially well suited to machine learning, data science, and education. So, if you want to execute torch code, then go for Colab.

**Comment by mrdbourke** [comment]
Hi @Darktechnodream ,

You can use PyCharm, yes. 

I'd check out their Jupyter Notebook support here: https://www.jetbrains.com/help/pycharm/jupyter-notebook-support.html (the course is taught with Google Colab, which is Google's version of Jupyter Notebooks)

You can also try VS Code, they have Jupyter Notebook support too: https://code.visualstudio.com/docs/datascience/jupyter-notebooks
